documentation/OpenAPI/example.rst

```diff
@@ -86,11 +88,17 @@ We need to configure Jane before generation. So we create a ``.jane-openapi`` fi
         'namespace' => 'CatFacts\Api',
         'directory' => __DIR__ . '/generated/',
         'date-format' => \DateTimeInterface::RFC3339_EXTENDED, // date-time format use by CatFact API
+        'client' => 'psr18',
     ];
 
 It will contains a reference to your main schema file (that file can be linked to other files if you want), the PHP
 namespace you want for generated classes and the directory you want to use.
 
+.. warning::
+
+    Client argument is recommended, if you don't fill it, it will generated a HTTPlug Client which is not what we are
```

Comment:
It's not using a HTTPlug client which is deprecated, it's only the generation of a client implementing HTTPlug client interface.
HTTPlug client extends PSR18 (which is fine) AND provides useful thing like the PluginClient for instance.